miio/vacuum.py

```diff
@@ -84,6 +84,13 @@ class FanspeedE2(enum.Enum):
     Turbo = 100
 
 
+class FanspeedS7(enum.Enum):
+    Quiet = 101
+    Balanced = 102
+    Turbo = 103
+    Max = 104
```

Comment:
Please use the similar naming as other vacuums above, the mid-term plan is to consolidate available fan speeds (to make it easier for homeassistant to support also non-rockrobo vacuums supported by this library).

Reply:
So what is called "turbo" in the app equals Medium and what is called "max" in the app equals Turbo? Maybe we should create some kind of mapping table in the documentation for this or it gets confusing at some point. But I fully agree with the consolidation

Reply:
After a second thought, I think this can stay for now as it is. I have some scribbles on a shared API and I'll create an issue to gather feedback on that later.

Anyways, I just took a look at the existing vacuum implementations for fan speeds:
Viomi: Silent, Standard, Medium, Turbo
Dreame: Quiet, Default, Medium, Strong
Rockrobo: Gentle (E2, v2 only?), Silent, Standard, Medium, Turbo

so it's quite a mess with names (and separate enums) but four different speeds is common so I think it'll be fairly simple to find a common base there when the time comes... :-)

Reply:
Didn't see your comment before changing it as suggested previously. But you are right, in the future it is just a matter of deciding which generic names to choose for the different speeds. In pretty much every case it is: silent, default, more, max